### ISSUE TITLE
Error on xxit() command

### ISSUE THREAD

**Issue by mnislam01** [pyodide/pyodide#1889]
![error](https://user-images.githubusercontent.com/24929466/137575626-d6d57a84-c836-47c4-9944-c015e8a0e226.png)

Getting error on exit() command in the emulator.

**Comment by hoodmane** [comment]
What is the expected behavior? I think this is a reasonable outcome of calling exit() since there is no actual way for Pyodide to exit.

**Comment by rth** [comment]
Yes, I agree that raising `SystemExit` is the [expected behavior](https://docs.python.org/2/library/constants.html#quit). Closing. If you see other things, don't hesitate to open other issues.
